api/dotnet/IndigoObject.cs

```diff
@@ -1694,6 +1694,11 @@ public string check2(string type)
             return dispatcher.checkResult(IndigoLib.indigoCheckObj2(self, type));
         }
 
+        public IndigoObject fingerprint()
```

Comment:
Do we have real demand for such method (except in test, where no problem to pass null)?